ruby/example/ThreatExchangeTest.rb

```diff
@@ -0,0 +1,45 @@
+#!/usr/bin/env ruby
+# ThreatExchange example script.
+require 'ThreatExchange'
+require_relative 'lib/example'
+access_token = 'abc123|somethingsomething'
+TEClient = ThreatExchange::Client.new(access_token)
+#------------------------------------------------------------------------------
+banner('Example Malware Analysis')
+query = {limit: 1000, text: 'bepush', strict_text: false}
```

Comment:
Because the default limit may change per the Platform API, it's probably better not to explicitly set a value here. Maybe offer it as an optional parameter. But we should use the system default by default. ;-)